src/agent/agent.c

```diff
@@ -551,12 +551,15 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
 
         if (configfile != NULL) {
                 result = cfg_load_from_file(agent->config, configfile);
-                if (result != 0) {
+                if (result < 0) {
                         fprintf(stderr,
                                 "Error loading configuration file '%s', error code '%s'.\n",
                                 configfile,
                                 strerror(-result));
                         return false;
+                } else if (result > 0) {
+                        fprintf(stderr, "Error parsing configuration file '%s' on line %d\n", configfile, result);
```

Comment:
Is it possible to check the length of the line?  Something like:
```
if (line_from_configuration contain length > MAX_LENGTH ) {  
      fprintf(stderr, "Achieved the max length in the configuration file, please see our man page for more information");
}
```

Reply:
No, unfortunately not. We just have the line number where some error has occurred.

Reply:
all good, thanks for all your answers!